.github/workflows/build.yml

```diff
@@ -4,25 +4,21 @@ jobs:
   build:
     strategy:
       matrix:
-        java-version: [1.8, 11, 14]
+        java-version: [8, 11, 14]
     runs-on: ubuntu-20.04
+    container:
+      image: openjdk:${{ matrix.java-version }}-jdk
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - uses: gradle/wrapper-validation-action@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.java-version }}
-      - uses: eskatos/gradle-command-action@v1.3.2
-        with:
-          gradle-version: wrapper
-          arguments: check build publishToMavenLocal --stacktrace --parallel
-          dependencies-cache-enabled: false
-          configuration-cache-enabled: false
+      - run: ./gradlew build publishToMavenLocal --stacktrace --parallel --no-daemon
```

Comment:
`--no-daemon` makes no diffrence here, its used once and then the whole container is thrown away.